rest-api/dao/participant_summary_dao.py

```diff
@@ -53,9 +53,21 @@
                    AND physical_measurements_status = :completed
                    AND samples_to_isolate_dna = :received)
              THEN :full_participant
+             WHEN (consent_for_study_enrollment = :submitted
+                   AND consent_for_electronic_health_records = :unset
+                   AND consent_for_dv_electronic_health_records_sharing = :submitted
```

Comment:
Alternatively, just OR above

Reply:
agreed, we've already seen that a participant may have both types of consent, and as far as I know, neither one takes precedence.

Reply:
done